modules/AlphaWalletFoundation/AlphaWalletFoundation/BlockscanChat/BlockscanChat.swift

```diff
@@ -25,6 +25,7 @@ public class BlockscanChat {
         infoLog("[BlockscanChat] Fetching unread count for \(address.eip55String)…")
         return networkService
             .dataTaskPublisher(GetUnreadCountEndpointRequest(address: address))
+            .receive(on: DispatchQueue.global())
```

Comment:
@oa-s Why? Is it because it's too slow?

Reply:
yes, especially its able to see when on initial app lanuch, when loading tickers and other calls